### PR TITLE
Docs: fix joinGet function

### PR DESCRIPTION
**Pull request by godfreyd**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category (leave one):
- Documentation

### CODE REVIEW

File: docs/en/query_language/functions/other_functions.md [hunk 1]
```diff
@@ -762,12 +762,68 @@ Takes the states of the aggregate function and returns a column with values, are
 For example, takes state of aggregate function (example runningAccumulate(uniqState(UserID))), and for each row of block, return result of aggregate function on merge of states of all previous rows and current row.
 So, result of function depends on partition of data to blocks and on order of data in block.
 
-## joinGet('join_storage_table_name', 'get_column', join_key) {#other_functions-joinget}
+## joinGet {#joinget}
 
-Gets data from [Join](../../operations/table_engines/join.md) tables using the specified join key.
+The function lets you extract data from the table the same way as from a [dictionary](https://clickhouse.yandex/docs/en/query_language/dicts/).
```
Comment (blinkov) [inline docs/en/query_language/functions/other_functions.md]: internal links have to be relative and point to markdown files

File: docs/en/query_language/functions/other_functions.md [hunk 2]
```diff
@@ -762,12 +762,68 @@ Takes the states of the aggregate function and returns a column with values, are
...
+
+- `join_storage_table_name` — name of the table where search will be performed.
+- `value_column` — name of the column of the table that contains required data.
+- `join_keys` — list of keys.
+
+**Returned value**
+
+Returns list of values corresponded to list of keys.
+
+If certain doesn't exist in source table then `0` or `null` will be returned based on [join_use_nulls](https://clickhouse.yandex/docs/en/operations/settings/settings/#settings-join_use_nulls) setting.
```
Comment (blinkov) [inline docs/en/query_language/functions/other_functions.md]:
```suggestion
If some keys doesn't exist in source table then default value or `NULL` will be returned (depending on [join_use_nulls](https://clickhouse.yandex/docs/en/operations/settings/settings/#settings-join_use_nulls) setting). 
```

**Review by blinkov — CHANGES_REQUESTED** [review]
also there's a broken docs test

**Comment by alexey-milovidov** [comment]
Was documented in newer PR.

**Pull request closed without merging**